gym_roboy/envs/ros_proxy.py

```diff
@@ -99,3 +125,28 @@ def read_state(self):
         future = self.step_client.call_async(req)
         self._wait_until_future_complete_or_timeout(future)
         return self._make_robot_state(future.result())
+
+    def forward_new_goal(self, goal_joint_angle):
+        assert len(goal_joint_angle) == 3
```

Comment:
3 is a magic number. Please refer to one of the existing class variables like `MsjROSProxy.DIM_JOINT_ANGLE`

Reply:
Replaced